CHANGELOG.md

```diff
@@ -23,6 +23,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Project generated by `snforge` contains `assert_macros` dependency with version 0.1.0 for Scarb <= 2.8.0, otherwise equal to Scarb
 - Support for overriding fork configuration in test attribute with a different block ID, tag, or hash.
+- `--no-optimization` flag that can be used to build contracts using the [starknet contract target](https://docs.swmansion.com/scarb/docs/extensions/starknet/contract-target.html#starknet-contract-target. This is the default behavior when using Scarb < 2.8.3
+
+### Changed
+
+- For Scarb >= `2.8.3` contract artifacts are built as part of the test target now. This process speeds up the compilation time. It can be disabled using the [--no-optimization flag](https://foundry-rs.github.io/starknet-foundry/appendix/snforge/test.html#--no-optimization)
```

Comment:
Maybe let's mention that this may result in behavior not being 100% consistent with the real network